drop/modules/aberrant-splicing-pipeline/fds_annotation.R

```diff
@@ -0,0 +1,67 @@
+### 20210604 klutz
+
+### basic annotations (start, end, none, both) for full fds
```

Comment:
Add a bit more details what the function is doing. Especially going through it you somehow only looking at the max expressed junctions? Why? Please describe this as a doc. For more details see roxygen and some bioconductor packages.